docs/external-template-creation.md

```diff
@@ -158,6 +158,8 @@ Once you have the story JSON, several code changes are needed to add it to the l
     import { default as template } from './template';
 
     export default {
+      // day this template was added to the codebase
+      creationDate: new Date(2021, 0, 1),
```

Comment:
Using `new Date()` is brittle because it uses your local computer's timezone and not the site's timezone.

I suggest using `toDate(date, getOptions())` from our date package.

Instead of importing that in every file, I would just use a plain string here instead. e.g. `2021-01-01T00:00:00.000Z`.

We can then turn these into proper dates using `toDate()` within `getTemplates()`